onadata/apps/logger/migrations/0017_remove_xform_sms.py

```diff
@@ -12,10 +12,18 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.AlterUniqueTogether(
-            name='xform',
-            unique_together={('user', 'id_string')},
-        ),
```

Comment:
Thanks for catching this. It seems like it's impossible to reverse this meaningfully because:
* `sms_id_string` was not nullable and defaulted to a default string;
* when we drop the column, we don't save its values anywhere;
* if we run the `RemoveField` backwards, it populates `sms_id_string` with an empty string for each `logger_xform` row;
* attempting to add the `unique_together` constraint fails for any user who has more than one form. For example, consider user 123 has two forms; the constraint would see `(123, '')` twice and fail with:
    ```
    django.db.utils.IntegrityError: could not create unique index "logger_xform_user_id_sms_id_string_97791222_uniq"
    DETAIL:  Key (user_id, sms_id_string)=(123, ) is duplicated.
    ```

Reply:
I agree that it won't help. The SMS feature has been hidden/broken for a while on kobocat anyway. Thank you for the explanation.